db/dbformat.h

```diff
@@ -145,9 +145,6 @@ inline void UnPackSequenceAndType(uint64_t packed, uint64_t* seq,
                                   ValueType* t) {
   *seq = packed >> 8;
   *t = static_cast<ValueType>(packed & 0xff);
-
-  assert(*seq <= kMaxSequenceNumber);
```

Comment:
This change is needed for testing corrupted memtable entries would see a `Status::Corruption` (otherwise they'd sometimes hit this assertion failure).

Reply:
assertions are useful code documentation. Consider leaving them as comments, like

    // Removed for corruption work: assert(*seq <= kMaxSequenceNumber);